shared/bazel/rules/cc_rules.bzl

```diff
@@ -400,3 +600,86 @@ def wpilib_cc_static_library(
         srcs = ["//:license_pkg_files", name + "-static.pkg"],
         tags = ["no-remote", "manual"],
     )
+
+def _generate_def_windows_impl(ctx):
+    """
```

Comment:
Can you describe the expected use/behavior of this on non-windows platforms? This rule implementation reads like it'll just run through and return an empty set of files in the providers, which is reasonably but non-intuitive.

Reply:
Comment improved in the def file review.